students/DanCornutt/session06/test_mailroom.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import os
+import pytest
+from mailroom import *
+
+
+def test_check_donor_1():
+    """Finds donor in list"""
+    assert check_donor("Billy Bills") is True
+
+def test_check_donor_2():
+    """Does not find donor in list"""
+    assert check_donor("Billy Billz") is False
+
+def test_gen_letter_1():
+    """Generates valid letter for donor in list"""
+    assert gen_letter("Billy Bills") == """Dearest Billy Bills,
```

Comment:
this works, but will fail if the template changes at all.

can you test some of the contents of the letter, without checking the entire text?